pages/resources/profiles.md

```diff
@@ -314,33 +314,33 @@ In summary the _PARADISEC profile_ covers as of
     -   `description`: `Text`
 
 This is a good example of how a specific profile can guide a user interface and
-[extend RO-Crate](1.1/appendix/jsonld.html#extending-ro-crate) with additional terms.
+[extend RO-Crate](/specification/1.1/appendix/jsonld#extending-ro-crate) with additional terms.
 
 _Note that some RO-Crates in the [Modern PARADISEC demonstrator](https://mod.paradisec.org.au/) have
 evolved from this profile to conform with RO-Crate 1.1's
-[repository content types](1.1/provenance.html#digital-library-and-repository-content)_
+[repository content types](/specification/1.1/provenance#digital-library-and-repository-content)_
 
 ## ARC RO-Crate profile
 
-A profile of RO-Crate for [Annotated Research Contexts](https://nfdi4plants.org/content/learn-more/annotated-research-context.html) (ARC), developed by [DataPLANT](https://nfdi4plants.org/).
-An ARC consists of [ISA](https://isa-specs.readthedocs.io/en/latest/isamodel.html) metadata describing the experimental setup and computational workflows given in CWL.
-The current [profile](https://github.com/nfdi4plants/ARC-specification/blob/main/ARC%20specification.md#appendix-conversion-of-arcs-to-ro-crates) requires the crate to follow the ISA Investigation profile on the top level.
+A profile of RO-Crate for [Annotated Research Contexts](https://nfdi4plants.org/content/learn-more/annotated-research-context) (ARC), developed by [DataPLANT](https://nfdi4plants.org/).
+An ARC consists of [ISA](https://isa-specs.readthedocs.io/en/latest/isamodel) metadata describing the experimental setup and computational workflows given in CWL.
+The current [profile](https://github.com/nfdi4plants/ARC-specification/blob/main/ARC%20specification#appendix-conversion-of-arcs-to-ro-crates) requires the crate to follow the ISA Investigation profile on the top level.
```

Comment:
Sorry for lack of clarity - only _internal_ links needed `.html`/`.md` removed. In most cases it doesn't matter for external links, but for github links to `.md` files the suffix needs to remain
```suggestion
The current [profile](https://github.com/nfdi4plants/ARC-specification/blob/main/ARC%20specification.md#appendix-conversion-of-arcs-to-ro-crates) requires the crate to follow the ISA Investigation profile on the top level.
```

Reply:
ok, tested most of the links but will make another pass through in more detail and get this fixed.